src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -48,6 +48,7 @@ class TelemetrySession(object):  # pylint: disable=too-many-instance-attributes
     payload_properties = None
     exceptions = []
     module_correlation = None
+    extension_name = 'None'
```

Comment:
Why are some Nones in quotes and others not?

Reply:
I didn't guard the `None` at the place this property is used.